spec/capabilities.md

```diff
@@ -197,7 +197,15 @@ The `with` field of the invocation must be set to the DID of the memory space to
 
 #### Caveats
 
-None currently, but this is expected to change once pagination is fully implemented.
+`cursor` can be set to start listing from an item in the middle of the list. Its value should be a `cursor` returned by a previous invocation of `store/list`
+`size` can be set to change the number of items returned by an `store/list` invocation
+
+Taken together, the CARs in the `shards` array should contain all the blocks in the DAG identified by the `root` CID.
```

Comment:
Does this make sense for `store/list`? afaict `shards` and `root` are only defined on caps in the `upload/` namespace?